identifier-ui/src/Auth.js

```diff
@@ -14,7 +14,7 @@ const Auth = ({onSuccess, onFailure}) => {
     };
     var authenticationDetails = new cognito.AuthenticationDetails(authenticationData);
     var userData = {
-      Username : 'EthanTheMan',
+      Username : authenticationData.Username,
```

Comment:
we can just use the `username` state variable here
```suggestion
      Username : username,
```